docs/docs/config/power.md

```diff
@@ -18,11 +18,12 @@ In the deep sleep state, the keyboard additionally disconnects from Bluetooth an
 
 Definition file: [zmk/app/Kconfig](https://github.com/zmkfirmware/zmk/blob/main/app/Kconfig)
 
-| Config                          | Type | Description                                           | Default |
-| ------------------------------- | ---- | ----------------------------------------------------- | ------- |
-| `CONFIG_ZMK_IDLE_TIMEOUT`       | int  | Milliseconds of inactivity before entering idle state | 30000   |
-| `CONFIG_ZMK_SLEEP`              | bool | Enable deep sleep support                             | n       |
-| `CONFIG_ZMK_IDLE_SLEEP_TIMEOUT` | int  | Milliseconds of inactivity before entering deep sleep | 900000  |
+| Config                                 | Type | Description                                           | Default |
+| -------------------------------------- | ---- | ----------------------------------------------------- | ------- |
+| `CONFIG_ZMK_IDLE_TIMEOUT`              | int  | Milliseconds of inactivity before entering idle state | 30000   |
+| `CONFIG_ZMK_SLEEP`                     | bool | Enable deep sleep support                             | n       |
+| `CONFIG_ZMK_IDLE_SLEEP_TIMEOUT`        | int  | Milliseconds of inactivity before entering deep sleep | 900000  |
+| `CONFIG_ZMK_AWAKE_WHILE_BLE_CONNECTED` | bool | Whether to stay awake while BLE is connected          | n       |
```

Comment:
I notice that CONFIG_ZMK_AWAKE_WHILE_BLE_CONNECTED stands out due to its deviation from the consistent naming convention where the other options include "SLEEP."

Considering consistency aids clarity and contributes to a better user experience, I propose renaming `CONFIG_ZMK_AWAKE_WHILE_BLE_CONNECTED` to `CONFIG_ZMK_NO_SLEEP_WHILE_BLE_CONNECTED`

As description I would suggest `Prevent deep sleep while BLE is connected`

Reply:
That was actually the first name i used, but pete suggested changing it.  See [here](https://discord.com/channels/719497620560543766/719544230497878116/1218308284801351852)

Im fine with any name as the relevant part is its functionality... Will change it to whichever is prefered.

Reply:
This should be updated to describe the behavior for peripheral as well.